package.json

```diff
@@ -38,11 +38,11 @@
     "end-to-end",
     "e2e"
   ],
-  "author": "Kent C. Dodds <kent@doddsfamily.us> (http://kentcdodds.com/)",
+  "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.8.4",
-    "@testing-library/dom": "^6.15.0",
+    "@testing-library/dom": "beta",
```

Comment:
Curious what this will do? This is being merged into `master`, but uses the `beta` npm tag for `@testing-library/dom`?